sw/device/lib/dif/dif_template.h.tpl

```diff
@@ -2,32 +2,29 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
```

Comment:
very minor nit: The `.tpl` suffix already implies that this is a template. So you could call this file `dif_ip.h.tpl` instead. (I am fine either way though).

Reply:
I think we should keep it as it already is, since documentation already refers to it by this name in other places.